Static site generators

miajim opened the pull request:
### Description

Created a new entry for Static Site Generators in general concepts. Closes #548 .

### Type of Change

- Adding a new entry
- 
### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Codecademy/docs?pullRequest=797) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=797) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=797) it.</sub>

SSwiniarski commented on content/general/concepts/static-site-generators/static-site-generators.md:
```diff
@@ -0,0 +1,38 @@
+
+## Examples of SSGs
+
+Some commonly used SSGs include Jekyll, Hugo, Gatsby, and [Next.js](https://www.codecademy.com/resources/docs/general/next-js). Visit [staticgen.com](http://staticgen.com) for a comprehensive list of SSGs available in different languages.
```
```suggestion
Some commonly used SSGs include Jekyll, Hugo, Gatsby, and [Next.js](https://www.codecademy.com/resources/docs/general/next-js). A comprehensive list of SSGs available in different languages is available at [staticgen.com](http://staticgen.com).
```

Dusch4593 reviewed (changes requested):
@miajim Thank you so much for this new entry! Excited for this one 😄 I'm finished with my review and left you a few comments/suggestions. Let me know if you have and thoughts or questions; happy to help!

SSwiniarski commented on content/general/concepts/static-site-generators/static-site-generators.md:
```diff
@@ -0,0 +1,38 @@
+  - 'Web Development'
+Tags:
+  - 'Static Site'
+  - 'Static Site Generator'
```
```suggestion
```

SSwiniarski commented on documentation/tags.md:
```diff
@@ -236,6 +237,8 @@ Specificity
 SQLite
 Stacks
+Static Site
+Static Site Generator
```
```suggestion
```

SSwiniarski reviewed (changes requested):
@miajim, I did a first review and made some suggestions.

SSwiniarski commented:
Actually, a second review. Me and @Dusch4593 appear to have been reviewing this simultaneously.

miajim reviewed (commented):
Done! Thanks @Dusch4593 and @SSwiniarski for taking the time to review!

Dusch4593 commented:
Thank you so much @miajim for your work on this awesome new entry! It is now live! 🎉 👍🏻 

https://www.codecademy.com/resources/docs/general/static-site-generators

I noticed something, though. It looks like your profile isn't showing up on the contributor list on the right?

<img width="881" alt="Screen Shot 2022-06-16 at 2 50 22 PM" src="https://user-images.githubusercontent.com/23144845/174143905-b50ceaa3-a408-4641-b044-1c47a4f05ce1.png">

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"
6. 
Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. 😄

miajim commented:
You're right it doesn't look like my profile is showing up, thanks for pointing that out. I recently added my GH username a few days ago to my profile (without the leading `@`) so I figured it would show up automatically.

Dusch4593 commented:
Looks like a profile has appeared! Is this you?

<img width="917" alt="Screen Shot 2022-06-17 at 8 31 18 AM" src="https://user-images.githubusercontent.com/23144845/174298719-c489c6b1-fecc-43e5-b97f-9b7dcbaefea5.png">

miajim commented:
Yep! I changed `@web3031029486` to a different username and added a profile picture this morning, so maybe within the next 24 hours it'll reflect the changes.

Dusch4593 commented:
Awesome, thanks!
